utilities/patchOnUsBundles/script.js

```diff
@@ -0,0 +1,48 @@
+import { CosmosClient } from "@azure/cosmos";
+
+const key = "<cosmos-key>";
+const endpoint = "<cosmos-endpoint>";
+const databaseId = "db";
+
+const client = new CosmosClient({ endpoint, key });
+const { database } = await client.databases.createIfNotExists({ id: databaseId });
+
+const CONTAINER_ID_BUNDLES = "bundles";
+const CONTAINER_ID_VALID_BUNDLES = "validbundles";
+const CONTAINER_ID_ARCHIVED_BUNDLES = "archivedbundles";
+
+const ONUS_SUFFIX = "_ONUS";
+
+async function patchObjects(container, bundles) {
+    const operations = bundles.map(bundle => ({
+        "operationType": "Patch",
+        "id": bundle.id,
+        "partitionKey": bundle.idPsp,
+        "resourceBody": { operations: [{ op: 'add', path: '/onus', value: bundle.name.includes(ONUS_SUFFIX) || bundle.idChannel.includes(ONUS_SUFFIX)}] }
```

Comment:
The onUs condition is evaluated only on the suffix idChannel, is it posssible to modify the condition when assigning the value? (using the second clause only and with `endsWith` type selection instead of includes)